docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
++--------------------------+-----------------------------+-----------------------+
+| history_kind             | :ref:`historyqospolicykind` | KEEP_LAST             |
++--------------------------+-----------------------------+-----------------------+
+| history_depth            | int32_t                     | 1                     |
++--------------------------+-----------------------------+-----------------------+
+| max_samples              | int32_t                     | -1 (Length Unlimited) |
++--------------------------+-----------------------------+-----------------------+
+| max_instances            | int32_t                     | -1 (Length Unlimited) |
++--------------------------+-----------------------------+-----------------------+
+| max_samples_per_instance | int32_t                     | -1 (Length Unlimited) |
++--------------------------+-----------------------------+-----------------------+
+
+* **Service cleanup delay**: It controls when the service can remove all the information regarding a data instance. That
+  information is kept until the following conditions are met:
+
+  * The instance has been explicitly disposed and its InstanceState becomes `NOT_ALIVE_DISPOSED`.
+  * There isn't any alive DataWriter writing the instance, which means that all existing writers either unregister the
+    instance or lose their liveliness.
+  * A time interval longer than the one established on the service_cleanup_delay has elapsed since the moment the
+    service detected that the two previous conditions were met.
+
+* **History kind**: Controls the kind of the :ref:`historyqospolicy` associated with the Durability Service fictitious
+  entities.
+* **History depth**: Controls the depth of the :ref:`historyqospolicy` associated with the Durability Service fictitious
+  entities.
+* **Max samples**: Controls the maximum number of samples of the :ref:`resourcelimitsqospolicy` associated with the
+  Durability Service fictitious entities.
+  This value must be higher than the maximum number of samples per instance.
+* **Max instances**: Controls the maximum number of instances of the :ref:`resourcelimitsqospolicy` associated with the
+  Durability Service fictitious entities.
+* **Max samples per instance**: Controls the maximum number of samples within an instance of the
+  :ref:`resourcelimitsqospolicy` associated with the Durability Service fictitious entities.
+  This value must be lower than the maximum number of samples.
+
+.. note::
+     This QoS Policy concerns to Topic and DataWriter entities.
+     :raw-html:`<br />`
+     It cannot be changed after the creation of the entities.
+
+.. _entityfactoryqospolicy:
+
+EntityFactoryQosPolicy
+^^^^^^^^^^^^^^^^^^^^^^
+
+This QoS Policy controls the behavior of an :ref:`dds_layer_core_entity` when it acts as a factory for other entities.
+As you know, the entities can be created either enabled or disabled, a fact that can be controlled using this QoS
+Policy.
```

Comment:
Does not give any information that is not covered on the previous and following sentence. Remove it to simplify.